lib/quoters/WebhookQuoter.ts

```diff
@@ -119,68 +132,71 @@ export class WebhookQuoter implements Quoter {
         MetricLoggerUnit.Milliseconds
       );
 
-      this.log.info(
-        { response: hookResponse.data, status: hookResponse.status },
-        `Raw webhook response from: ${endpoint}`
-      );
+      const rawResponse = {
+        status: hookResponse.status,
+        data: hookResponse.data,
+        responseTime: Date.now(),
+        latencyMs: Date.now() - before,
+      };
 
       const { response, validation } = QuoteResponse.fromRFQ(request, hookResponse.data, request.type);
 
       // RFQ provider explicitly elected not to quote
       if (isNonQuote(request, hookResponse, response)) {
         metric.putMetric(Metric.RFQ_NON_QUOTE, 1, MetricLoggerUnit.Count);
         metric.putMetric(metricContext(Metric.RFQ_NON_QUOTE, name), 1, MetricLoggerUnit.Count);
-        this.log.info(
-          {
-            response: hookResponse.data,
-            responseStatus: hookResponse.status,
+        await this.firehose.sendAnalyticsEvent({
```

Comment:
One question here, do you have any suspicions on how this will effect overall latency? With our old logs we didn't have to `await` so they didn't effect our overall endpoint response time but looks like these might change that a little bit?